lim_test.py

```diff
@@ -0,0 +1,259 @@
+'''tests for the longintmath.py module'''
+import unittest
+import longintmath as lim
+
+class TestLIMFunc(unittest.TestCase):
+    '''tests for the long_int_(mathfunction) functions'''
+    def setUp(self):
+        self.long_big = 10**1000
+        self.float_big = 1.0e+300
+        self.float_sm = 1.0e-100
+
+    def tearDown(self):
+        del self.long_big
+        del self.float_big
+        del self.float_sm
+
+    def error_tester(self, result, expected):
+        '''use the inverse error to avoid trouble with overflow errors.'''
+        allowed_error = 1e-8
+        if result - expected == 0:
+            inv_error = float('inf')
+        else:
+            inv_error = abs(expected // (result - expected))
+        self.assertGreater(inv_error, 1/allowed_error)
+
+    def test_li_div(self):
+        '''test long_int_div'''
+        self.assertEqual(lim.long_int_div(self.float_big, self.long_big), 0)
+        self.assertEqual(lim.long_int_div(self.float_sm, self.float_big), 0)
+
+        result1 = lim.long_int_div(self.float_big, self.float_sm)
```

Comment:
what different cases are these testing? When you revisit this code in a year will you remember what result1-4 are representing? 

These should be test methods where the method name describes why it exists. Do not be afraid on long test method names, they're not real methods (no one calls them). They're glorified comments.